pkg/core/block/header.go

```diff
@@ -28,6 +30,9 @@ type Header struct {
 	// The height of the block must be exactly equal to the height of the previous block plus 1.
 	Timestamp uint64
 
+	// Nonce is block random number.
+	Nonce uint64
```

Comment:
Along with this change we need to adjust `Block` structure that used in our interop package. We also need to adjust `BlockToStackItem` of the native Ledger contract. In C# standard serialisation mechanisms are used for this purpose, thus Nonce is also included in the result of `ledger.GetBlock`.

Reply:
Fixed.